Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts

```diff
@@ -347,7 +347,7 @@ namespace gdjs {
     }
 
     setFlow(flow: number, tank: number): void {
-      this.emitter.frequency = flow < 0 ? 0.0001 : 1.0 / flow;
+      this.emitter.frequency = flow < 0 ? 0 : 1.0 / flow;
```

Comment:
why was it not 0 before?

Reply:
I guess, the emission was considered as ended and the emitter was deleted right from the start.

Reply:
Or the frequency cannot take the value 0 in pixi?